packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -21,6 +27,7 @@ import SettingsPage from './SettingsPage.svelte';
       message="Install an authentication provider extension to add an authentication provider here."
       hidden="{$authenticationProviders.length > 0}" />
     {#each $authenticationProviders as provider}
+      {@const sessionRequests = provider.sessionRequests || []}
```

Comment:
```suggestion
      {@const sessionRequests = provider.sessionRequests ?? []}
```

Reply:
There is no difference in || or ?? here. It guards against the null (which never happens) and the undefined when there is no sessions. It is a matter of style.

Reply:
see https://github.com/containers/podman-desktop/issues/6374
And
https://typescript-eslint.io/rules/prefer-nullish-coalescing/

Reply:
this rule IMO is questionable, it has too many options to create exceptions form the rule. Anyway, it is not relevant for this code, so it is updated to ??.